### DISCUSSION TITLE
Can I pianificate theme switching and not just the dark mode?

### DISCUSSION THREAD

**Discussion by PeopleInside** [AutoDarkMode/Windows-Auto-Night-Mode#482] [Q&A] [answered]
Hi,
what this app do is activate the dark mode. I want to switch to a Windows 11 dark theme at night and have a light daily time at day.

Theme will change also desktop background and is more nice than just dark mode.
Is a feature that actually this app doesn't support?
There is a feature request for that, has sense to made one or is already possible?

Thank you!

**Comment by Spiritreader** [comment, marked as answer]
You can do that by using the Windows theme mode (via the personalization tab) and then select your day theme for night and vice versa
